src/Tribe/Editor/Configuration.php

```diff
@@ -15,6 +16,45 @@ public function hook() {
 		add_filter( 'tribe_editor_config', array( $this, 'editor_config' ) );
 	}
 
+	/**
+	 * Setup the initial Event Object.
+	 *
+	 * @since TBD
+	 *
+	 * @todo @backend Improve this method and verify if this is the correct place for this to live.
+	 *
+	 * @param null|int|WP_Post $event
+	 *
+	 * @return array Setup the Data for the Event Post Object.
+	 */
+	public function get_event_object( $event = null ) {
+		$data = [
+			'is_new_post' => true,
+		];
+
+		if ( ! $event ) {
+			$event = tribe_get_request_var( 'post', false );
+		}
+
+		if ( $event ) {
+			$event = get_post( $event );
+		}
+
+		if ( ! $event instanceof  WP_Post) {
```

Comment:
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: Expected 1 space after "instanceof"; 2 found
* :no_entry_sign: No space before closing parenthesis is prohibited
* :no_entry_sign: No space before closing parenthesis is prohibited

<sub>hash: 5a349b4f160ef1c36a39ebd239e36bfe</sub>